### ISSUE TITLE
MetaDataLoader shuffle default to true

### ISSUE THREAD

**Issue by bkj** [tristandeleu/pytorch-meta#4]
Hi -- 

I'm curious what you think about changing the default `shuffle` parameter in `MetaDataLoader` to `true`. 

I ran
```
dataset     = omniglot("./data", meta_split='train', ways=20, shots=1, shuffle=True)
dataloader = BatchMetaDataLoader(dataset, batch_size=16, num_workers=4)
```
and was getting too-good results.  It took a bit of time to track down that `BatchMetaDataLoader` outputs classes in `combinations(range(num_classes), num_shots)` order, so the batches are super correlated w/ one another.  I can't really imagine a use-case where you'd want this ordering of your batches.  Thoughts?

Thanks!
~ Ben

**Comment by tristandeleu** [comment]
I think that's very reasonable, I'm all for it, to avoid any confusion. The reason behind this default value was that [PyTorch's DataLoader](https://github.com/pytorch/pytorch/blob/082f195ac9ec7e2ee9b1d25329b357cb2cedd6f3/torch/utils/data/dataloader.py#L75-L76) has a default value of `shuffle=False`. However while this might not be too much of a issue in standard learning, this definitely is in meta-learning (and especially the way the tasks are created). I will update the default value, thanks!
